src/current/cockroachcloud/cloud-api.md

```diff
@@ -331,9 +320,7 @@ curl --request POST \
         "{region_name}": 3
       },
       "hardware": {
-        "machine_spec": {
-          "machine_type": "{machine_type}"
-        }
+        "num_virtual_cpus": {num_cpus}
```

Comment:
The `num_virtual_cpus` needs to be within the `machine_spec` section.

Reply:
Thanks. Fixing via https://github.com/cockroachdb/docs/pull/18951; ready for review.